Sources/Sentry/SentryCrashIntegration.m

```diff
@@ -216,13 +202,22 @@ + (void)enrichScope:(SentryScope *)scope
 
 #endif
 
-    NSDictionary *systemInfo = [SentryCrashIntegration systemInfo];
-    [osData setValue:systemInfo[@"osVersion"] forKey:@"build"];
-    [osData setValue:systemInfo[@"kernelVersion"] forKey:@"kernel_version"];
-    [osData setValue:systemInfo[@"isJailbroken"] forKey:@"rooted"];
+    NSDictionary *systemInfo = [crashWrapper systemInfo];
+
+    // This only applies if SentryCrashIntegration is not installed
+    if (systemInfo != nil && systemInfo.count != 0) {
+        [osData setValue:systemInfo[@"osVersion"] forKey:@"build"];
+        [osData setValue:systemInfo[@"kernelVersion"] forKey:@"kernel_version"];
+        [osData setValue:systemInfo[@"isJailbroken"] forKey:@"rooted"];
+    }
 
     [scope setContextValue:osData forKey:@"os"];
 
+    // This only applies if SentryCrashIntegration is not installed
+    if (systemInfo == nil || systemInfo.count == 0) {
```

Comment:
Are one of these comments inverted? The conditions they appear above are...

Reply:
Good catch. I changed the comment. Thanks.